### DISCUSSION TITLE
Audio Script Error

### DISCUSSION THREAD

**Discussion by hockeygoalie35** [RandomNinjaAtk/arr-scripts#25] [Q&A] [answered]
Hi, I get an error while the audio script runs:

>  2023-07-17 10:56:48 :: Audio :: 2.1 :: 4
> 2023-07-17 10:56:49 :: Audio :: 2.1 :: 3
> 2023-07-17 10:56:50 :: Audio :: 2.1 :: 2
> 2023-07-17 10:56:51 :: Audio :: 2.1 :: 1
> 2023-07-17 10:56:52 :: Audio :: 2.1 :: To kill script, use the following command:
> 2023-07-17 10:56:52 :: Audio :: 2.1 :: kill -9
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Add Deezer Top Artists is disabled (enable by setting addDeezerTopArtists=true)
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Add Deezer Top Album Artists is disabled (enable by setting addDeezerTopAlbumArtists=true)
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Add Deezer Top Track Artists is disabled (enable by setting addDeezerTopTrackArtists=true)
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Add Deezer Related Artists is disabled (enable by setting addRelatedArtists=true)
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Download Location: /config/extended/downloads/audio
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Output format: native
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Output bitrate:
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Download Quality Check Disabled (enable by setting: requireQuality=true
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Preferred audio lyric type: both
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Tidal Country Code set to: US
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Replaygain Tagging Enabled
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Match Distance: 5
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Beets Tagging Enabled
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: Beets Matching Threshold 90%
> 2023-07-17 10:56:54 :: Audio :: 2.1 :: DEEZER :: Verifying deemix configuration
> 2023-07-17 10:56:55 :: Audio :: 2.1 :: DEEZER :: ARL Token: Configured
> 2023-07-17 10:56:55 :: Audio :: 2.1 :: DEEZER :: Configuring deemix client
> wget: bad port spec '127.0.0.1:'
> wget: bad port spec '127.0.0.1:'
> 2023-07-17 10:56:55 :: Audio :: 2.1 :: FINDING MISSING ALBUMS :: sorted by date
> 2023-07-17 10:56:55 :: Audio :: 2.1 ::  Missing Albums Found!
> 2023-07-17 10:56:55 :: Audio :: 2.1 :: Getting Missing Album IDs
> ./run: line 998: [: -ge: unary operator expected
> wget: bad port spec '127.0.0.1:'
> 2023-07-17 10:56:55 :: Audio :: 2.1 :: FINDING CUTOFF ALBUMS sorted by date
> 2023-07-17 10:56:55 :: Audio :: 2.1 ::  CutOff Albums Found Found!
> 2023-07-17 10:56:55 :: Audio :: 2.1 :: Getting CutOff Album IDs
> ./run: line 1035: [: -ge: unary operator expected
> 2023-07-17 10:56:55 :: Audio :: 2.1 :: Script end...
> 2023-07-17 10:56:55 :: Audio :: 2.1 :: Script sleeping for 2m...

**Comment by RandomNinjaAtk** [comment]
Did you modify/change the default port that lidarr uses? If so, that is your problem.

**Comment by hockeygoalie35** [comment, marked as answer]
I got it, I was using an old version of Lidarr (9/30).
